backend/mongodb.js

```diff
@@ -62,3 +55,34 @@ app.get("/", async (req, res) => {
     res.status(500).send("Error fetching movies");
   }
 });
+app.get('/comments', async (req, res) => {
+  try {
+
+      const comments = await MovieComment.find(); 
+      
+      return res.json(comments); 
+  } catch (error) {
+    console.error('Error fetching comments:', error);
+    res.status(500).send('Error fetching comments');
+  }
+});
+
+
+app.post('/comments', async (req, res) => {
+  try {
+      const { comment } = req.body;
+      if (!comment || comment.trim() === "") {
+        return res.status(400).json({ message: "Please enter some comment" });
```

Comment:
do you need this check here?
like do you need to check whether the comment has been provided and if we have added actual text in comment, shouldn't the commentSchema be able to validate the comment?